src/test/checkdatasig_tests.cpp

```diff
@@ -140,6 +140,8 @@ BOOST_AUTO_TEST_CASE(checkdatasig_test) {
     MMIXLinearCongruentialGenerator lcg;
     for (int i = 0; i < 4096; i++) {
         uint32_t flags = lcg.next();
+        // Flag cannot occur in script execution, only in Taproot key spend
+        flags &= ~SCRIPT_REQUIRE_TAPROOT_SIGNATURES;
```

Comment:
Seems like there should be a test that this fails?

Reply:
It's tested in e.g. sigencoding_tests.cpp, so here it seemed superfluous, especially since it's against the "spirit" of the test.